linchpin/provision/roles/openstack/tasks/provision_os_volume.yml

```diff
@@ -41,7 +41,7 @@
     auth: "{{ auth_var | omit_filter(omit) }}"
     state: "{{ vol.3 }}"
     size: "{{ vol.2 }}"
-    display_name: "{{ vol.1 }}_{{ vol.4 }}"
+    display_name: "{{ vol.1 }}default_delimiter{{ vol.4 }}"
```

Comment:
@skatlapa According to jinja template syntax the default_delimiter should be in {{  }}
else it will be taken as a string. 
Have you tested this piece?